domain/src/main/java/tachiyomi/domain/manga/model/Manga.kt

```diff
@@ -29,6 +29,9 @@ data class Manga(
     val initialized: Boolean,
     val lastModifiedAt: Long,
     val favoriteModifiedAt: Long?,
+    val version: Long,
+    val isSyncing: Long,
+
```

Comment:
Here

Reply:
I can't remove those two, I will get error when mapping the manga or chapter? in MangaRepositoryImpl and Chapter, since the model is not 1:1? 

I mean I could fix it by adding dummy or unused param of isSyncing with a long type but that's counter intuitive and the model should be 1:1 to what the db have. 

```prolog
Type mismatch.
Required:
(Long, Long, String, String?, String?, String?, List<String>?, String, Long, String?, Boolean, Long?, Long?, Boolean, Long, Long, Long, Long, UpdateStrategy, Long, Long, Long?, Long, Long) → TypeVariable(T)
Found:
KFunction23<Long, Long, String, String?, String?, String?, List<String>?, String, Long, String?, Boolean, Long?, Long?, Boolean, Long, Long, Long, Long, UpdateStrategy, Long, Long, Long?, Long, Manga>
```

Reply:
In that case might as well have a separate table for this stuff since it's needed but not necessarily as part of manga 🤔

Reply:
> In that case might as well have a separate table for this stuff since it's needed but not necessarily as part of manga/chapter 🤔

Hmm that will make it more complex, and it's part of it technically. Since we want to disable the trigger for that particular manga or chapter, like we still want it to work for the other ones while it's syncing if the user like reads other manga or chapter that's not being synced the trigger should still work. 

It's the same thing with the `LastFavoritedAt` field which we only use to check if a manga has ever been Favorited even once that way we don't sync junk stuff like users searches etc, keeps the data small etc. Besides that we are not using it anywhere else.

Reply:
`LastFavoritedAt` is used for sorting too. Also pretty sure you can do a join inside a trigger. If that doesn't work let's just go with a unused parameter route

Reply:
Okay done.